app/controllers/development_metrics_controller.rb

```diff
@@ -66,6 +77,38 @@ def build_metrics_definitions
     @pull_request_size_definition = MetricDefinition.find_by(code: :pull_request_size)
   end
 
+  def show_metrics?
+    metrics_to_show if metric_params[:name].present?
+
+    product_action = action == PRODUCTS_ACTION
+
+    @show_defect_escape_rate = product_action || @show_defect_escape_rate ? true : false
```

Comment:
This line looks confusing, I think we can do `@show_defect_escape_rate ||= product_action`